changelog/@unreleased/pr-137.v2.yml

```diff
@@ -0,0 +1,6 @@
+type: break
+break:
+  description: Fix Gradle 7 compatibility. Increase minimum required Gradle version
+    to 7.0.
```

Comment:
It's super unfortunate we need to split the versions like this. I think we might technically be able to support both by doing something like shading two different versions of the shadow plugin, or by having two different artifacts for this plugin (one <7 one >=7) but both are pretty grim.

I think with the excavator we can just choose to either keep the version pinned if `GradleVersionNumber.current()` is <7 or use latest if >7. People will just have to upgrade to 7 if they want a bug fix.